Source/WebInspectorUI/UserInterface/Base/Setting.js

```diff
@@ -198,6 +198,7 @@ WI.settings = {
     enableControlFlowProfiler: new WI.Setting("enable-control-flow-profiler", false),
     enableElementsTabIndependentStylesDetailsSidebarPanel: new WI.Setting("elements-tab-independent-styles-details-panel", true),
     enableLineWrapping: new WI.Setting("enable-line-wrapping", true),
+    enableNarrowLayout: new WI.Setting("enable-narrow-layout-mode", true),
```

Comment:
NIT: please match the name of the variable with the name of the setting, so change one of `"enable-narrow-layout"` or `enableNarrowLayoutMode`